app/routes/articles/components/ArticleEditor.tsx

```diff
@@ -189,15 +190,15 @@ const onSubmit = (
           cover: data.cover,
         }
       : {}),
-    ...normalizeObjectPermissions(data),
+    ...normalizeObjectPermissions({ ...data, currentUser: currentUser }),
```

Comment:
You shouldn't do this. It is more than enough that you set the default to the current user. If they remove themselves from the authors field, but it sets them as the author anyways that would be very strange.

Reply:
Is this not just to ensure there's always an author? If they remove themselves, and no author has been specified, it should default to `currentUser`, no? However, if they remove themselves and add someone else, it should just be that other user. I feel like that's the "better" MMI.

Reply:
Made the field required with front-end validation. An empty array seems to be allowed in the back-end but unless someone explicitly sends a request this should not happen.

` authors: data.authors.length > 0 ? data.authors.map((e) => e.value) : null,`